.github/dependabot.yml

```diff
@@ -0,0 +1,50 @@
+version: 2
+updates:
+  - package-ecosystem: 'terraform'
+    pull-request-branch-name:
+      separator: "-"
+    directory: "/terraform/account"
+    schedule:
+      interval: 'daily'
+  - package-ecosystem: 'terraform'
+    pull-request-branch-name:
+      separator: "-"
+    directory: "/terraform/environment"
+    schedule:
+      interval: 'daily'
+  - package-ecosystem: 'gomod'
+    pull-request-branch-name:
+      separator: "-"
+    directory: '/'
+    schedule:
+      interval: 'daily'
+  - package-ecosystem: 'composer'
+    pull-request-branch-name:
+      separator: "-"
+    directory: '/'
+    schedule:
+      interval: 'daily'
```

Comment:
This looks like its PHP related - hopefully not going to need that in this repo 🙏

Reply:
yeah, I've left all ecosystems in place on purpose

Reply:
removed :D